pvoutput/pvoutput.py

```diff
@@ -720,10 +799,13 @@ def download_multiple_systems_to_disk(
                 output_filename, pv_system_id, start_date, end_date
             )
 
-            # How much data is actually available?
-            date_ranges_to_download = self._filter_date_range(
-                output_filename, pv_system_id, date_ranges_to_download, min_data_availability
-            )
+            # # How much data is actually available?
```

Comment:
Might be good to put this back in, and add a comment saying, 

`if you dont have access to XXX API route, then comment this bit out` 

Or something like that